packages/fetchai/skills/ml_train/behaviours.py

```diff
@@ -64,8 +64,7 @@ def act(self) -> None:
             if not result.successful():
                 return
 
-            ml_task = result.get()
-            strategy.weights = ml_task.result
```

Comment:
So the property result does not need to be used?

Reply:
Yep